<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	property-alligment: 'AlignVCenter | AlignRight';
    border: 2px solid gray;
    background-color: rgb(215, 215, 215);
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>  0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multi">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 151, 87);
    border: 1px solid black;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 151, 87);
    border: 1px solid black;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 151, 87);
    border: 1px solid black;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 215, 215);
    border: 1px solid orange;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 151, 87);
    border: 1px solid black;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
  background-color: rgb(255, 255, 255);
    border: 1px solid green;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 215, 215);
    border: 1px solid orange;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dem">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 215, 215);
    border: 1px solid orange;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 151, 87);
    border: 1px solid black;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_null">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(215, 215, 215);
    border: 1px solid orange;
}

QPushButton:pressed{
	ackground-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    background-color: rgb(215, 51, 87);
    border: 1px solid black;
}

</string>
    </property>
    <property name="text">
     <string>RUssian gray Calculator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_extra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
